packages/network-controller/src/NetworkController.ts

```diff
@@ -431,12 +432,17 @@ export class NetworkController extends BaseController<
     | AutoManagedNetworkClient<CustomNetworkClientConfiguration>
     | AutoManagedNetworkClient<InfuraNetworkClientConfiguration>;
 
-  constructor({
-    messenger,
-    state,
-    infuraProjectId,
-    trackMetaMetricsEvent,
-  }: NetworkControllerOptions) {
+  #log: Logger | undefined;
+
+  constructor(
+    {
+      messenger,
+      state,
+      infuraProjectId,
+      trackMetaMetricsEvent,
+    }: NetworkControllerOptions,
+    log?: Logger,
```

Comment:
I think we still want this to be in the set of options and not a separate argument, yes @Gudahtt?